JobRecruiting/JobRecruiting/src/main/webapp/WEB-INF/tags/ifHasRole.tag

```diff
@@ -0,0 +1,16 @@
+<%-- 
+    Document   : ifHasRole
+    Created on : Jan 14, 2021, 9:42:42 PM
+    Author     : DENISA
+--%>
+
+<%@tag description="put the tag description here" pageEncoding="UTF-8"%>
+
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+
+<%@attribute name="role"%>
+
+<c:if test="${pageContext.request.isUserInRole(role)}" >
```

Comment:
Si aici la fel, daca zici de CandidateRole, merge pe oricare. 
Vezi ca si mie mi-a trebuit ceva de genul in servletul Interviews.java.
Poate te ajuta...